tests/test_toolchain.py

```diff
@@ -2,6 +2,7 @@
 import sys
 import pytest
 import mock
+from util import backup_attributes
```

Comment:
I'm not sure if the name you give to the file can cause troubles in some import situations...since we already have a file with the same name in `pythonforandroid.util` :thinking:...I would give it another filename to avoid strange situations...but maybe there is no need...what do you think about this?

Reply:
Yes it also bugged me. Actually I usually call it `utils.py`, then I changed it to be consistent. But still yes it also bugs me that it looks the same and one is for tests only. I don't have preferences, tell me how you like it to be called and I can call it that way :smile:

Reply:
- `tools`
  - `utilities`
  - `assets`
  ...

Reply:
What about `test_utils.py`?

Reply:
@JonasT, I would avoid `test_utils`...I think that unittests looks into the first part of the filename to determine if a file is a test or not...am I right?

Plus the mentioned file is not a test...it's a set of utilities to help us testing...I think that it's confusing...

Reply:
`utils_for_testing.py`? :smile:

Reply:
That looks better :wink: